### PR TITLE
Test Fixes –– All Tests Passing

### PR DESCRIPTION
**Pull request by DiosAkhilleus**

### CODE REVIEW

File: solana/program_v2/programs/gateway_v2/src/instructions/gatekeeper/verify_pass.rs [hunk 1]
```diff
@@ -51,6 +51,7 @@ pub struct PassVerify<'info> {
     #[account(
     seeds = [PASS_SEED, pass.subject.as_ref(), network.key().as_ref(), & pass.pass_number.to_le_bytes()],
     bump,
+    // TODO!: Constraint is incorrect here
```
Comment (JulianSpring) [inline solana/program_v2/programs/gateway_v2/src/instructions/gatekeeper/verify_pass.rs:54]: Lets add todos with a ticket

File: solana/program_v2/packages/tests/src/pass-suite/verify.test.ts [hunk 2]
```diff
@@ -64,7 +70,7 @@ describe('Verify a pass', () => {
         gatekeeperAuthority.publicKey,
         mintAccount.publicKey,
         gatekeeperPDA,
-        3000
+        10000
```
Comment (JulianSpring) [inline solana/program_v2/packages/tests/src/pass-suite/verify.test.ts:73]: Is this change required?

File: solana/program_v2/packages/tests/src/pass-suite/change_pass_gatekeeper.test.ts [hunk 3]
```diff
@@ -98,27 +109,78 @@ describe('Change pass gatekeeper', () => {
     expect(pass?.gatekeeper.toBase58()).to.equal(newDataAcct);
   });
 
-  it.skip('Cannot change to gatekeeper within a different network', async () => {
+  it('Cannot change to gatekeeper within a different network', async () => {
     // Assemble
-    const atlNetworkAuthority = Keypair.generate();
-    const [altStakingAccount] = await NetworkService.createStakingAddress(
-      atlNetworkAuthority.publicKey
+    const newAdminAuthority = Keypair.generate();
```
Comment (JulianSpring) [inline solana/program_v2/packages/tests/src/pass-suite/change_pass_gatekeeper.test.ts:114]: You can reuse this function to do the set up :) `setUpAdminNetworkGatekeeper`

File: solana/program_v2/packages/tests/src/pass-suite/issue.test.ts [hunk 4]
```diff
@@ -75,13 +75,14 @@ describe('issue', () => {
...
-        gatekeeperAta.address,
         networkAta.address,
+        gatekeeperAta.address,
         funderAta.address,
         funderKeypair.publicKey
       )
       .withPartialSigners(funderKeypair)
-      .rpc();
+      .rpc()
+      .catch((e) => console.log(e));
```
[the remark this answers is not in the document]
Comment (DiosAkhilleus) [reply solana/program_v2/packages/tests/src/pass-suite/issue.test.ts:85]: Good catch, thank you

File: solana/program_v2/packages/tests/src/pass-suite/verify.test.ts (hunk 2 above)
Comment (DiosAkhilleus) [reply solana/program_v2/packages/tests/src/pass-suite/verify.test.ts:73]: Changed for testing, forgot to switch back. Thanks for catching

File: solana/program_v2/programs/gateway_v2/src/instructions/gatekeeper/verify_pass.rs (hunk 1 above)
Comment (DiosAkhilleus) [reply solana/program_v2/programs/gateway_v2/src/instructions/gatekeeper/verify_pass.rs:54]: Done

File: solana/program_v2/packages/tests/src/pass-suite/change_pass_gatekeeper.test.ts (hunk 3 above)
Comment (DiosAkhilleus) [reply solana/program_v2/packages/tests/src/pass-suite/change_pass_gatekeeper.test.ts:114]: Will attempt to do so!

Comment (DiosAkhilleus) [reply solana/program_v2/packages/tests/src/pass-suite/change_pass_gatekeeper.test.ts]: Done

**Review by JulianSpring — APPROVED** [review]
